rmm/armv9a/src/exception/lower/synchronous/sys_reg.rs

```diff
@@ -120,15 +149,21 @@ fn handle_sysreg_id(vcpu: &mut VCPU<Context>, esr: u64) -> u64 {
         ISS_ID_AA64PFR0_EL1 => unsafe { ID_AA64PFR0_EL1.get() & mask },
```

Comment:
```suggestion
ISS_ID_AA64PFR0_EL1 => { ID_AA64PFR0_EL1.get() & mask },
```

I think we can eliminate a number of "unsafe" blocks that access to system registers, by putting "unsafe" inside the function of `define_sys_register! get()`, like below.
```
pub fn get(&self) -> u64 {
        let rtn;
        unsafe {
                    core::arch::asm! {
                        concat!("mrs {}, ", stringify!($regname)),
                        out(reg) rtn
                    }
           }
           rtn
 }
```
It would look more neat but I don't think this change is necessary. (i.e., whether to change or not is up to you-:)

Reply:
I read some comments in a paper about the 'unsafe' block: it is perferred and safer that exposing unsafe to the function definition rather than internally digesting it inside a function so as for the caller to handler properly. Based on that, exposing unsafe at get() looks better. However, I don't think the current caller in sys_regs.rs handles it properly. I hope we can disscuss how to handle the unsafe blocks after CC summit.

Reply:
> it is perferred and safer that exposing unsafe to the function definition

totally makes sense